tests/test_wave_01.py

```diff
@@ -143,12 +141,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert MOVIE_TITLE_1 in updated_data["watched"][1]["title"]
```

Comment:
Nice! By using `in`, one might think that `updated_data["watched"][1]["title"]` is a collection (list, set, tuple, ...). Instead `==` might be a more suitable choice.